### DISCUSSION TITLE
How to export the result from "pcmci.print_significant_links" or Lag function Matrix

### DISCUSSION THREAD

**Discussion by JOHNPAUL-ADIMS** [jakobrunge/tigramite#255] [Q&A]
Hi, 

Please, how best can I export t the result of the` Lag function Matrix` or ` pcmci.print_significant_links` to txt? 

Running an analysis, and I want to extract the following results for all my time series. I have 26-time series data and am comparing this with another 6-time series data. Want to be able to save the following:

1. Time lag
2. Auto-correlation values of each significant node
3. Cross-correlation values of each significant links

_from the picture below, this is where the information is stored._
<img width="607" alt="image" src="https://user-images.githubusercontent.com/57724518/201213200-87e2671a-18ec-472b-8601-5c9dee617a75.png">

Please, can someone help me out with this?

**Comment by jakobrunge** [comment]
Use the new ``write_csv`` function in the plotting module.
